indra/newview/llimprocessing.cpp

```diff
@@ -451,6 +452,30 @@ void LLIMProcessing::processNewMessage(LLUUID from_id,
     bool is_linden = chat.mSourceType != CHAT_SOURCE_OBJECT &&
         LLMuteList::isLinden(name);
 
+    /***
+    * The simulator has flagged this sender as a bot, if the viewer would like to display
+    * the chat text in a different color or font, the below code is how the viewer can
+    * tell if the sender is a bot.
+    *-----------------------------------------------------
+    bool is_bot = false;
+    if (metadata.has("sender"))
+    {   // The server has identified this sender as a bot.
+        is_bot = metadata["sender"]["bot"].asBoolean();
+    }
+    *-----------------------------------------------------
+    */
```

Comment:
The comment seems incorrect to assume the the sender has been flagged as a bot.  For more correctness please indicate the server may or may not have flagged...

"The simulator has flagged this sender as a bot..." --> "The simulator may flag this sender as a bot in 'metadata'.

Reply:
"The simulator **may have** flagged...